pkg/commands/process/settings/rules/javascript/express/path_traversal.yml

```diff
@@ -8,6 +8,9 @@ patterns:
           - resolve
```

Comment:
Please add a comment for the types of `req` that are considered for User Input in the `javascript_express_path_traversal_request_obj` pattern.

Reply:
The `req` types that we consider User Input are the following: 
https://github.com/Bearer/bearer/blob/d6e2dbe928007211e8f02b9b376b9457d63327c6/pkg/commands/process/settings/rules/javascript/express/path_traversal.yml#L12-L18
I feel the patterns are descriptive enough themselves. What sort of comment would you like to see here?